scripts/west_commands/sign.py

```diff
@@ -425,17 +425,19 @@ def sign(self, command, build_dir, bcfg, formats):
         board = cache['CACHED_BOARD']
         log.inf('Signing for board ' + board)
         target = self.edt_get_rimage_target(board)
-        log.inf('Signing for SOC target ' + target)
+        conf = target + '.toml'
+        log.inf('Signing for SOC target ' + target + ' using ' + conf)
 
         if not args.quiet:
             log.inf('Signing with tool {}'.format(tool_path))
 
         bootloader = str(b / 'zephyr' / 'bootloader.elf.mod')
         kernel = str(b / 'zephyr' / 'zephyr.elf.mod')
         out_bin = str(b / 'zephyr' / 'zephyr.ri')
+        conf_path = str(b / '../modules/audio/sof/rimage/config' / conf)
```

Comment:
Seems like the configuration for signing ought to be part of the SoC integration and in the Zephyr tree, no?  I know with SOF there's always going to be a tension about what parts of configuration are "application" and what parts are "platform".  But with rimage it seems like the answer should be simple: rimage is the tool, the configuration lives elsewhere.

Also solves the problem about hardcoding the path. :)

Reply:
that would seem logical to me too, yes, but that would have to be discussed with rimage developers. Maybe they want to keep them under their control for the case if they have to change anything in their configuration handling, so that they don't have to care about versioning...